.github/workflows/nightly.yaml

```diff
@@ -106,17 +106,30 @@ jobs:
       - name: Configuring SeaLights
         if: ${{ steps.operator-image-existence-checker.outputs.OPERATOR_IMAGE_EXISTS == 'true' }}
         run: |
-          echo "[SeaLights] Configuring SeaLights to scan for main branch nightly"
-          ./slcli config create-bsid --app rhdh-operator --branch ${{ matrix.branch }} --build "$BUILD_TIME:$LATEST_COMMIT_SHA"
+          echo "[SeaLights] Configuring SeaLights to scan for ${{ matrix.branch }} branch nightly"
```

Comment:
Do we really need every night scanning for released branches?

Reply:
IMO, yes, as long as they are supported.